runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpStateSingle.java

```diff
@@ -254,6 +355,7 @@ XpSnapshotSingle snapshot()
 			.actionsInSession(actions)
 			.actionsRemainingToGoal(getActionsRemaining())
```

Comment:
Actions remaining and actions/hr should simply return kills remaining and kills/hr when needed. same with actions in session.